pkgs/by-name/be/better-commits/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs 
+, nix-update-script
+}:
+
+buildNpmPackage rec {
+  pname = "better-commits";
+  version = "1.16.1";
+
+  src = fetchFromGitHub {
+    owner = "Everduin94";
+    repo = pname;
```

Comment:
I'm not a big fan of getting the repo from pname, but that's just a personal nit